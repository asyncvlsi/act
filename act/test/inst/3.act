defproc foo(bool a, b) ;

bool p;

foo f(p);
