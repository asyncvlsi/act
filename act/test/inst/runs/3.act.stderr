ERROR: File `3.act', line 5, col 6
	Mismatch in number of port parameters (1 v/s actual 2)
	Instance type: -global-::foo
	Prototype is: defproc <> (bool a; bool b)
